sccm/develop/reference/core/clients/deploy/sms_clientbaseline-sever-wmi-class.md

```diff
@@ -1,114 +1,3 @@
 ---
-title: "SMS_ClientBaseline Class"
-titleSuffix: "Configuration Manager"
-ms.date: "09/20/2016"
-ms.prod: "configuration-manager"
-ms.technology: configmgr-sdk
-ms.topic: conceptual
-ms.assetid: 524d8fd8-d226-4cb3-85d5-91ecfc45eef7
-author: aczechowski
-ms.author: aaroncz
-manager: dougeby
+redirect_url: ./sms_clientbaseline-server-wmi-class

```

Comment:
Hi @exchange12rocks, one of the reviewers tried testing the redirect and it's not working. The theory is that it's because this old file was changed as well as deleted. Can you back out this addition to the file, so that the end state is it's just deleted.

Reply:
@aczechowski Done!